sample/src/main/res/values/colors.xml

```diff
@@ -6,7 +6,12 @@
     <color name="purple_700">#FF3700B3</color>
     <color name="teal_200">#FF03DAC5</color>
     <color name="teal_700">#FF018786</color>
-    <color name="black">#FF000000</color>
+
+    <color name="black1">#FF000000</color>
+    <color name="black2">#000000</color>
+    <color name="black3">#F000</color>
+    <color name="black4">#000</color>
```

Comment:
instead of putting these in the normal colors file, we can move them into a separate file called `duplicate_colors_sample.xml` with a comment highlighting what is this meant for

Reply:
Done.